guide/src/class.md

```diff
@@ -16,6 +16,68 @@ struct MyClass {
 
 The above example generates implementations for `PyTypeInfo` and `PyTypeObject` for `MyClass`.
 
+## Get Python objects from `pyclass`
+In rust side, we can get a Python object which includes `pyclass` by 3 methods.
```

Comment:
This needs a short explanation that if you instantiate a rust struct, it doesn't need to be part of a python object a python object. PyRef however guarantees that the struct instance is part of a python object and that you're also holding the GIL, which allows you to modify the struct. 

Everything else looks good!

Reply:
:+1: